src/ferc_eia_match/name_cleaner.py

```diff
@@ -0,0 +1,335 @@
+"""This module contains the implementation of CompanyNameCleaner class from OS-Climate's financial-entity-cleaner package."""
```

Comment:
This is a modified version of the `financial-entity-cleaner` package that only works for US English language companies.

The license is Apache Software License, Other/Proprietary License (Apache 2), which I think means it's okay  to vendor in this way?